lib/modules/apostrophe-palette-widgets/views/macros.html

```diff
@@ -8,4 +8,12 @@
       }
     }) }}
   </div>
+{% endmacro %}
+
+{% macro stylesheetLink(global) %}
+  <link rel="stylesheet" href="/modules/apostrophe-global/palette-styles?{{ global.paletteCounter or 'e' }}">
```

Comment:
Great direction, but use a named query parameter like "cb=" so it's easier to go in the direction of using other query parameters too later if you want to.